src/pages/Wishlist.js

```diff
@@ -26,32 +20,46 @@ export default function Wishlist() {
       addToCart({ ...selectedItem, quantity: 1 });
     };
   };
-  const handleRemoveWishlist=selectedItemId=>{
-return ()=>{
-  removeFromWishlist(selectedItemId)
-}
-  }
+  const handleRemoveWishlist = (selectedItemId) => {
+    return () => {
+      removeFromWishlist(selectedItemId);
+    };
+  };
   return (
     <main>
       <div className="content-wrapper column">
-        {data.length>0&&<h3 className="text-center">
-          My Wishlist <span className="light-txt">{`(${wishlistData.length})`}</span>
-        </h3>}
-        {data.length>0?
-        <section id="wishlist" className="wishlist-wrapper">
-        {data.map(item=><Card item={item} key={item.id} isWishlist={true} onWishlistClick={handleRemoveWishlist(item.id)}               onAddToCartClick={handleAddToCart(item)}
-/>)}
-        </section>
-        :
-        <div className="flex-center column mt-16">
-            <img src="images/wishlist_empty.png" alt="empty wishlist" loading="lazy" />
+        {data.length > 0 && (
+          <h3 className="text-center">
+            My Wishlist{" "}
+            <span className="light-txt">{`(${wishlistData.length})`}</span>
+          </h3>
+        )}
+        {data.length > 0 ? (
+          <section id="wishlist" className="wishlist-wrapper">
+            {data.map((item) => (
+              <Card
+                item={item}
+                key={item.id}
+                isWishlist={true}
+                // onWishlistClick={handleRemoveWishlist(item.id)}
+                // onAddToCartClick={handleAddToCart(item)}
```

Comment:
Can remove this commented code

Reply:
Actually I have kept it because it will be used once I add the other logic